rapids_pytest_benchmark/rapids_pytest_benchmark/plugin.py

```diff
@@ -472,6 +473,7 @@ def pytest_sessionfinish(session, exitstatus):
         commitTime = asvMetadata.get("commitTime", commitTime)
         commitRepo = asvMetadata.get("commitRepo", commitRepo)
         commitBranch = asvMetadata.get("commitBranch", commitBranch)
+        requirements = json.loads(asvMetadata.get("requirements")) if "requirements" in asvMetadata else None
```

Comment:
I would do this to clean it up a little:
```
requirements = json.loads(asvMetadata.get("requirements", "{}"))
```
This is safe since `asvdb` turns a `None` into a `{}` internally anyway.
Also, if we can assume the string is python syntax instead of JSON (which are almost identical), we can then just do:
```
requirements = eval(asvMetadata.get("requirements", "{}"))
```
But it might be easiest to tell users simply that all args to `--benchmark-asv-metadata` are either python strings or JSON, but probably not some of each. Plus whatever we decide can be used to parse the strings, so we should probably just pick one.